sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_models.py

```diff
@@ -175,15 +173,55 @@ class KeyVaultSetting(object):
     """A Key Vault setting.
 
     :ivar str name: The name of the account setting.
-    :ivar str value: The value of the pool setting.
-    :ivar SettingType type: The type specifier of the value.
+    :ivar value: The value of the account setting.
+    :vartype value: str or bool
```

Comment:
The instance variable will always be type `str` right?
It's just the constructor parameter that could be `bool` or `str`?